package.json

```diff
@@ -166,7 +166,7 @@
     "micromatch": "^4.0.7",
     "minimist": "^1.2.8",
     "nanoid": "^5.0.7",
-    "npm-run-all": "^4.1.5",
```

Comment:
Just a note, but pnpm supports this out of the box. We are using this in Vitest for example to run both several scripts at the same time:

```ts
pnpm run --stream '/^dev:/'
```

Reply:
Thanks, updated!